### ISSUE TITLE
Running into ImportError

### ISSUE THREAD

**Issue by mike3085** [caarmen/pylint-json2checkstyle#5]
Hello, I am hitting the following import error either by running `pylint-json2checkstyle` or `pylint --load-plugins=pylint_json2checkstyle.checkstyle_reporter --output-format=checkstyle`

Using Python 3.11.0 and pylint 3.0.2 on Windows x64

```
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "C:\Users\Mike\AppData\Local\Programs\Python\Python311\Scripts\pylint-json2checkstyle.exe\__main__.py", line 4, in <module>
  File "C:\Users\Mike\AppData\Local\Programs\Python\Python311\Lib\site-packages\pylint_json2checkstyle\checkstyle_reporter.py", line 11, in <module>
    from pylint.interfaces import IReporter
ImportError: cannot import name 'IReporter' from 'pylint.interfaces' (C:\Users\Mike\AppData\Local\Programs\Python\Python311\Lib\site-packages\pylint\interfaces.py)
```

**Comment by caarmen** [comment]
Thanks for bringing this to my attention! I can reproduce this issue.

It seems to occur using pylint 3.x. I can't reproduce it with pylint 2.17.7.

I'll post an update here when I'm able to push a fix.

**Comment by caarmen** [comment]
This should be fixed in 0.0.13.
